eip.tf

```diff
@@ -1,6 +1,41 @@
+locals {
+  # if eip map not defined at all, want_eip is false.
+
+  # want EIP on the box at the end of this entire process?
+  want_eip = lookup(var.eip, "create", null) != null
```

Comment:
I don't want to be too pedantic, but we have `want_eip` here and `eip_create` and `eip_name` elsewhere. If we care about consistency, this variable should be `eip_wanted` but I won't dig in my heels on this point.